product_development.md

```diff
@@ -0,0 +1,96 @@
+
+A bug ticket should have an average resolution rate of less than a week.
+
+Bugs may be closed or converted into a story, if there is no longer a high urgency and no impact on the stability of the platform. Updates on bug tickets, especially closure without resolution, need to be communicated with product proactively.
+
+### Tech debt
+
+Engineering leads are responsible for the stability of the platform and the development velocity, and consequently for when and what technical debt needs to be handled.
+
+Work on technical debt must always be proactively communicated with the product team and documented in a ticket. Tech debt tickets must be pulled into the backlog in a refinement meeting, as they bind resources and take up significant space in the engineering backlog.
+
+### Platform announcement Slack channel
+
+Product, Design, Engineering, and Content team members post every change to go live on the platform into the Slack channel [#platform-announcements](https://voiio.slack.com/archives/C02BXK6R16V). This channel serves as a changelog for all employees.
+
+## Meetings & Collaboration
+
+### Agile principles
+
+* Make work visible
+* Meaningful & actionable conversations
+* Decisions are made, where the work is done
+* Regularly reflect on how we work together
+
+### Stakeholder Meetings
+
+Stakeholder meetings are held weekly to discuss the product roadmap, upcoming features, and any blockers that need to be resolved or questions that need to be answered. It is the best path to discuss new ideas and to align on the product's direction. The product manager will relate work to the OKR progress and stakeholders will come prepared with issues/ideas and data, adding their topics to this document: [Stakeholder Meeting Platform](https://docs.google.com/document/d/1uhX68whx9T56unUlEUdyjUdq_dOJpHwO7Ycv892hqA8/edit)
+
+The meeting is led by the Product Manager and attended by key stakeholders and an engineering lead.
+
+### Refinement Meetings
+
+Refinement meetings are held weekly and are the handover point from the Product Team to the Engineering Team. The product manager presents problems, opportunities, and priorities to the Engineering Team. Engineers ask questions and either pull the issue into the backlog or push it back to the product team for further refinement.
+
+The meeting is led by the product manager and attended by the product and engineering team.
+
+### Standups
+
+Standups are held daily and are the heartbeat of the team (Engineering, Content, Product). They are used to update the team on the progress of ongoing tasks and blockers. Standups are not for problem-solving, but for identifying issues that need to be solved later during the day.
+
+Standups are led by the respective Team Lead going through the Kanban board and attended by the entire team and, optionally, by people who have been closely working with the respective team.
```

Comment:
A daily being led by someone is not a practice we follow. I'm not against it but we should reflect reality here.

Reply:
That's OK as long as they work. However, if we loose discipline someone would need to facilitate them and be accountable. I wouldn't stress about it for now and this description isn't only for engineering dailies.